common/lib/xmodule/xmodule/js/fixtures/problem_content.html

```diff
@@ -23,7 +23,7 @@ <h3 class="hd hd-2 problem-header">Problem Header</h3>
         <button class="show btn-default btn-small"><span class="show-label">Show Answer(s)</span> <span class="sr">(for question(s) above - adjacent to each field)</span></button>
       </span>
     </div>
-    <button class="check btn-brand" data-checking="Submitting" data-value="Submit"><span class="check-label">Submit</span><span class="sr"> your answer</span></button>
+    <button class="submit Submit btn-brand" data-submitting="Submitting" data-value="Submit"><span class="submit-label">Submit</span><span class="sr"> your answer</span></button>
```

Comment:
What is the class "Submit" for? We don't typically have uppercase classnames (and there is also a class of "submit").